### PR TITLE
Validator ultimate guide

### PR DESCRIPTION
**Pull request by SaveTheAles**
Available [here](https://github.com/cybercongress/congress/blob/validator-ultimate-guide/blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md)

### CODE REVIEW

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+docker run -d -p 9000:9000 -v /var/run/docker.sock:/var/run/docker.sock portainer/portainer
+```
+
+3. Open your browser and go to:
+
+```bash
+localhost:9000
+```
+
+![](portainer_start.jpg)
```
Comment (arturalbov) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: A little bit confusing. Replace with screenshot on `local` tab.

**Comment by asadovka** [comment]
NVIDIA 1 step:

correct command is
sudo add-apt-repository ppa:graphics-drivers/ppa

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
+# Ultimate cyberd validator setup. Testnet: Euler-1.
+
```
Comment (litvintech) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: Please add here the table with a difference between google and cyberd

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+Recommended requirements:
+
+```
+CPU: 6 cores
+RAM: 64 GB
+SSD: 512 GB
+Connection: 100Mb, Fiber, Stable and low-latency connection
+GPU: GeForce 1070-1080, CUDA
+Software: Docker, Ubuntu 16.04/18.04 LTS
+```
```
Comment (litvintech) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: @SaveTheAles Please provide more powerful description why GPU memory size matters

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+  --commission-rate="0.10" \
+  --commission-max-rate="0.20" \
+  --commission-max-change-rate="0.01" \
+  --chain-id=euler-1
+```
+
+#### Verify that you validating
+
+```bash
+docker exec -ti cyberd cyberdcli query stake validators --trust-node=true
```
Comment (litvintech) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: not stake, staking

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+docker run -d --name=cyberd --restart always --runtime=nvidia \
+ -p 26656:26656 -p 26657:26657 -p 26660:26660 \
+ -v /root/cyberd:/root/.cyberd \
+ -v /root/cyberdcli:/root/.cyberdcli \
+ cyberd/cyberd:euler-1
+```
+
+Don't forget to unjail if you was jailed during update.
+
+put more gpu
```
Comment (litvintech) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: describe this and next line briefly

File: blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md [hunk 6]
```diff
@@ -0,0 +1,447 @@
...
+sharding soon
+
+# Call to you, validator
+
+As at 2019/02/01 there 7 active validators. This is enough to support net, but this is not enough for quality grow. We looking for validators. Join us!
```
Comment (litvintech) [inline blog/drafts/ultimate-validator-guide/ultimate-validator-guide.md]: please provide more powerful call based on mission of cyberd protocol

**Pull request merged**